src/ansible_navigator/configuration_subsystem/definitions.py

```diff
@@ -131,7 +130,8 @@ def settings_file_path(self, prefix: str) -> str:
         return sfp.replace("_", "-")
 
 
-class SubCommand(SimpleNamespace):
+@dataclass(frozen=True)
+class SubCommand:
     """An object to hold a subcommand"""
 
     name: str
```

Comment:
Can we fix the `Union[None, str]` -> `Optional[str]` down on 139 while we're at it?

Reply:
Approved either way